source/Camera/InfoUtilOther/Warmup/Warmup.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+This example shows how to warm up the camera.
+*/
+
+#include <Zivid/Zivid.h>
+
+#include <chrono>
+#include <iostream>
+#include <thread>
+
+using HighResClock = std::chrono::high_resolution_clock;
+using Duration = std::chrono::nanoseconds;
+
+int main()
+{
+    try
+    {
+        Zivid::Application zivid;
+
+        std::cout << "Connecting to camera" << std::endl;
+        auto camera = zivid.connectCamera();
+
+        auto warmupTime = std::chrono::minutes(10);
+        auto captureCycle = std::chrono::seconds(5);
+        auto maxCaptureTime = std::chrono::milliseconds(1000);
+
+        std::cout << "Getting camera settings" << std::endl;
+        const auto parameters = Zivid::CaptureAssistant::SuggestSettingsParameters{
+            Zivid::CaptureAssistant::SuggestSettingsParameters::AmbientLightFrequency::none,
+            Zivid::CaptureAssistant::SuggestSettingsParameters::MaxCaptureTime{ maxCaptureTime }
+        };
+        const auto settings = Zivid::CaptureAssistant::suggestSettings(camera, parameters);
+
+        std::cout << "Warm-up time: " << warmupTime.count() << " minutes" << std::endl;
+
+        const auto beforeWarmup = HighResClock::now();
+
+        while(HighResClock::now() - beforeWarmup < warmupTime)
+        {
+            const auto beforeCapture = HighResClock::now();
+
+            camera.capture(settings);
+
+            const auto afterCapture = HighResClock::now();
```

Comment:
Not sure if beforeCapture and AfterCapture should be const? They change for each iteration. (Not sure what the standard is for const in loops)

Reply:
If they are const, then all variables in the loop could be const?

Reply:
Lets maybe ask SE? I do not know.